packages/amplication-notifications-service/src/contracts/kafkaMessagePayload.ts

```diff
@@ -0,0 +1,15 @@
+export class KafkaMessagePayload {
```

Comment:
What is the value of making this a class rather than an interface?

Reply:
@mshidlov would you mind sharing your input about it? I remember we talked about it when I built the `git-pull-service.`